### PR TITLE
Add dependency on libatomic1 library for Linux

### PR DESCRIPTION
**Pull request by McGiverGim**
Fixes https://github.com/betaflight/betaflight-configurator/issues/2933

I tested Ubuntu and seemed to work. I don't have now a way to test RPM part.

### CODE REVIEW

**Review by haslinghuis — DISMISSED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
Little update to:
- Upgrade the gulp-debian package to latest version
- Return when the rpmbuild or dpkg-deb utility is not found

**Review by haslinghuis — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
